lib/ca_testing/drivers/v4/browserstack/ios.rb

```diff
@@ -5,10 +5,11 @@ module Drivers
     module V4
       module Browserstack
         class Ios
-          attr_reader :ios_version
-          private :ios_version
+          attr_reader :device_name, :ios_version
+          private :device_name, :ios_version
```

Comment:
If its private, can you not just read the instance variable?

Reply:
Could do I guess. Think this is where I was asking a while ago about style. I've always used readers e.t.c. but can switch to just using the iVars proper.

Reply:
I don't mind to be honest, I hadn't seen that before, but its perfectly legit